### PR TITLE
update docs + travis + test-ci

### PR DESCRIPTION
**Pull request by puigmontella**
The following tutorials are incorporated:
- 1DWetAvalanche with dilatancy effects
- 2DWetAvalanche with dilatancy effects

A few changes in doc have been done to  improve style and correct typos

A new test (1DWetAvalanche with dilatancy effects) for the continuous integration has been added.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+
+sedfoam/.travis.yml
+@CyrilleBonamy
+CyrilleBonamy new css doc (#33)
+Latest commit 1ad0c8e 18 days ago
+History
+2 contributors
+@CyrilleBonamy
+@jchauchat
+100 lines (95 sloc) 3.5 KB
```
Comment (CyrilleBonamy) [inline .travis.yml:34]: What is that?
I think you can delete the 34 first lines.

File: .travis.yml [hunk 2]
```diff
@@ -98,3 +136,18 @@ notifications:
...
+    Status
+    Help
+    Contact GitHub
+    Pricing
+    API
+    Training
+    Blog
+    About
+
+
```
Comment (CyrilleBonamy) [inline .travis.yml:153]: i think you can delete the last lines (from  139 to 153)

File: test-ci/1DWetAvalanche/system/funkySetFieldsDict [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        }
+        Ub
+        {
+                field Ub;
+		expression "vector(0, 0, 0)";
+                keepPatches 1;
+        }
+ );
+
+// ************************************************************************* //
```
Comment (CyrilleBonamy) [inline test-ci/1DWetAvalanche/system/funkySetFieldsDict:40]: funkySetFieldsDict really needed?

File: test-ci/1DWetAvalanche/test_1DWetAvalanche.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
+import subprocess
+import numpy as np
+import fluidfoam
+from pylab import *
```
Comment (CyrilleBonamy) [inline test-ci/1DWetAvalanche/test_1DWetAvalanche.py:4]: i think we can delete this line

File: test-ci/1DWetAvalanche/test_1DWetAvalanche.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+import matplotlib.gridspec as gridspec
```
Comment (CyrilleBonamy) [inline test-ci/1DWetAvalanche/test_1DWetAvalanche.py:5]: we can delete this one too...

**Review by CyrilleBonamy — CHANGES_REQUESTED** [review]
Thank you very much for the hard work!

Please modify travis.yml, avoid funkySetFields if possible (codeStream is the solution), and do we need the  result "200" for test-ci?

File: .travis.yml (hunk 1 above)
Comment (puigmontella) [reply .travis.yml:34]: ok! I don't know how I messed it up!

File: .travis.yml (hunk 2 above)
Comment (puigmontella) [reply .travis.yml:153]: ok! I don't know how I messed it up!

File: test-ci/1DWetAvalanche/system/funkySetFieldsDict (hunk 3 above)
Comment (puigmontella) [reply test-ci/1DWetAvalanche/system/funkySetFieldsDict:40]: nope, deleting

File: test-ci/1DWetAvalanche/test_1DWetAvalanche.py (hunk 5 above)
Comment (puigmontella) [reply test-ci/1DWetAvalanche/test_1DWetAvalanche.py:5]: ok

File: test-ci/1DWetAvalanche/test_1DWetAvalanche.py (hunk 4 above)
Comment (puigmontella) [reply test-ci/1DWetAvalanche/test_1DWetAvalanche.py:4]: ok

**Pull request merged**